src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -109,8 +99,7 @@ public void assertContainsAny(@NotNull ItemStack item)
 	 * @param item   The itemstack to search for.
 	 * @param amount The minimum amount of items that one should have.
 	 */
-	public void assertContainsAtLeast(@NotNull ItemStack item, int amount)
-	{
+	public void assertContainsAtLeast(@NotNull ItemStack item, int amount) {
```

Comment:
You broke the formatting here.

```suggestion
	public void assertContainsAtLeast(@NotNull ItemStack item, int amount)
	{
```